library/src/scala/quoted/reflection/printers/Extractors.scala

```diff
@@ -1,22 +1,22 @@
-package scala.tasty
-package reflect
+package scala.quoted
+package reflection.printers
```

Comment:
`reflection` -> `reflect`?

If we have both names, it's easy to get confused and make mistakes. It would be nice to just use the same name through out.

Reply:
They have a different name to avoid confusion. I tried that but noticed that using `reflect` is more confusing because you would have `qctx.reflect` and `scala.quoted.reflect` in the same scope. It also clashes with `scala.reflect` which is annoying. `scala.quoted.reflection` is the one that will be used the less often hence the longer name.

Reply:
If it has to be different, what about choosing a name that does not contain `reflect*`?

Reply:
What name could be used? It should be clear that it is about reflection.

Reply:
I also wanted to distance the name from `tasty` to avoid confusion with the file format

Reply:
Can it be just `scala.quoted.printers`?

Reply:
We will also need to move `scala.tasty.Reflection` and `scala.tasty.reflect.*` into this package. 

```
scala.quoted
  - Expr, Type, ...
  - reflection (needs better name)
    - Reflection (or rename to Reflect to align with qctx.reflect)
    - printers
      - SourceCode, Extractors and SyntaxHighlighting
    - reflect (needs new name)
      - TreeMap, TreeAcumulator, TreeTraverser
```

Reply:
Yes, naming is always hard. What about `scala.quoted.trees` or `scala.quoted.ast`?

Reply:
I like `ast`, but now I realize that we may no even need to keep this logic in the library.

Reply:
Now I moved to printers out of the library. We never used them in the library anyway, only in the implementation of the `QuoteContext`. This will also allow us to rewrite them using the compiler trees directly for efficiency if we need it in the future.